BUILD.bazel

```diff
@@ -115,6 +115,7 @@ GA_LIBRARIES = [
 
 [cc_library(
     name = "experimental-{library}_mocks".format(library = library),
+    testonly = 1,
```

Comment:
nit: should these be `True`?

Reply:
On.  Ehh, I mean Yes.  Oops, I mean True.